app/models/publishing_pipeline_state.rb

```diff
@@ -166,7 +166,7 @@ def self.expire!(podcast)
   end
 
   def self.expire_pipelines!
-    Podcast.where(id: expired_pipelines.select(:podcast_id)).each do |podcast|
+    Podcast.with_deleted.where(id: expired_pipelines.select(:podcast_id)).each do |podcast|
```

Comment:
Has this happened?  We don't often delete podcasts in prod.

Reply:
In staging they pile up -- as part of CI